tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fvm_overview.md

```diff
@@ -0,0 +1,245 @@
+# New Feature: Finite Volume Method (FVM)
+
+!---
+
+## Advantages of Finite Volume Method
+
++(1) Flexible:+ Works with arbitrary cell types (even if they are mixed in the same problem)
+
++(2) Inherently conservative:+ Uses the integral form of the PDEs. This makes it perfect for
+      applications where conservation laws need to be respected (fluid flows).
+
++(3) "Accurate":+ Can be second-order accurate in space.
+
++(4) Rich literature:+ Widely used in Computational Fluid Dynamics (CFD) and has been researched for decades.
+
```

Comment:
I would add: (5) Robust: it is the current standard for most commercial (STAR-CCM+, ANSYS-Fluent, etc.) and major open-source (OpenFOAM, etc.) CFD codes

Reply:
Good idea! Added!